rpcs3/util/vm_native.cpp

```diff
@@ -135,7 +140,11 @@ namespace utils
 			size += 0x10000;
 		}
 
+#ifdef __APPLE__
+        auto ptr = ::mmap(use_addr, size, PROT_NONE, MAP_ANON | MAP_PRIVATE | MAP_JIT | c_map_noreserve, -1, 0);
```

Comment:
indentation

Reply:
I think you're using spaces instead of tabs, which is why it may appear as correct on your end

Reply:
Weird that only these lines were wrong. I replaced with tabs it looks good now.